src/vouchers.js

```diff
@@ -79,8 +84,12 @@ class RecrasVoucher {
     }
 
     changeTemplate(templateID) {
+        this.selectedTemplate = this.templates.filter(t => {
```

Comment:
Do these changes belong in this PR?

Reply:
Not necessarily, but I thought so when starting this branch

Reply:
okay, I think splitting PRs is usually a good idea, to keep it clear what the specific issue at hand is about.